### PR TITLE
fix: avoid dashboard icons shrinking on resize

### PR DESCRIPTION
**Pull request by ignromanov**
## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/domains/wallet/components/WalletsGroup/WalletsGroup.blocks.tsx [hunk 1]
```diff
@@ -72,14 +72,15 @@ export const GroupNetworkIcon: React.VFC<WalletsGroupNetworkIconProperties> = ({
...
-				"flex w-11 h-11 rounded-xl justify-center items-center bg-clip-padding hover:(border-theme-background shadow-xl)",
+				"flex basis-11 shrink-0 h-11 rounded-xl justify-center items-center bg-clip-padding hover:(border-theme-background shadow-xl)",
 				{
 					"bg-theme-background text-theme-secondary-700 ring-2 ring-theme-secondary-800": isDarkMode,
 					"bg-theme-secondary-100 text-theme-secondary-100": !isDarkMode,
 					"group-hover:ring-theme-secondary-700 group-hover:bg-theme-secondary-800":
 						isWalletsCollapsed && isDarkMode,
 				},
 			)}
+			style={{ flexBasis: "2.75rem", flexShrink: 0 }}
```
Comment (alesmit) [inline src/domains/wallet/components/WalletsGroup/WalletsGroup.blocks.tsx:83]: Is there any reason to apply the same properties on both style and class? TW classes `basis-11 shrink-0` do already what this style does

Comment (ignromanov) [reply src/domains/wallet/components/WalletsGroup/WalletsGroup.blocks.tsx]: Because of `flexBasis` is not working in TW2
I removed unused styles

**Comment by faustbrian** [comment]
@ignromanov conflicts

**Review by faustbrian — DISMISSED** [review]

**Comment by faustbrian** [comment]
@ignromanov failing build

**Comment by faustbrian** [comment]
@ignromanov failing build

**Review by faustbrian — APPROVED** [review]

**Pull request merged**